.travis.yml

```diff
@@ -0,0 +1,21 @@
+dist: trusty
+language: csharp
+solution: Banshee.sln
+
+addons:
+  apt:
+    packages:
+      - libgtk3.0-cil
+      - libdbus2.0-cil
+      - libtaglib2.1-cil
+      - libmono-addins0.2-cil
+      - libdbus-glib2.0-cil
+      - libgudev3.0-cil
+      - libmono-zeroconf1.0-cil
+      - libgconf2.0-cil
+env:
+  - BUILD_CONFIGURATION=Release
+  - BUILD_CONFIGURATION=Debug
+
+script:
+  - msbuild /p:Configuration=$BUILD_CONFIGURATION Banshee.sln
```

Comment:
Yeah I skimmed the build log last week but haven't had a chance to pick it apart - nice to see the successful build.

I've just checked the solution configuration on `master` branch and it looks like we need to enable some projects for full coverage:

- Backends\B.GStreamerSharp; (I think we only build the native one)
- Dap\B.AppleDevice;
- Dap\B.Karma;
- Extensions\B.BooScript;
- Extensions\B.SoundMenu; (I'm not on Ubuntu but it should probably build as long as it's still a thing)
- Extensions\B.SqlDebugConsole;
- Extensions\B.UbuntuOneMusicStore; (I thought UbuntuOne was dead? - think we can ignore this one)
- Extensions\B.UPnPClient;
- Extensions\B.YouTube.